querydsl-apt/src/main/java/com/querydsl/apt/DefaultConfiguration.java

```diff
@@ -13,28 +13,63 @@
  */
 package com.querydsl.apt;
 
-import static com.querydsl.apt.APTOptions.*;
-
-import java.lang.annotation.Annotation;
-import java.util.*;
-
-import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
-import javax.annotation.processing.RoundEnvironment;
-import javax.lang.model.element.*;
-import javax.lang.model.type.TypeMirror;
-
 import com.google.common.base.Function;
 import com.google.common.base.Splitter;
 import com.google.common.base.Strings;
 import com.mysema.codegen.model.ClassType;
-import com.querydsl.codegen.*;
+import com.querydsl.codegen.CodegenModule;
```

Comment:
why all these package refactorings?
still to the necessary changes

Reply:
Can we merge it as is? As it's been a while without update.